tests/cephfs/snapshot_clone/cross_platform_snaps.py

```diff
@@ -0,0 +1,154 @@
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Test Cases Covered:
+    CEPH-83573418	Create a Snapshot, reboot the node and rollback the snapshot
+
+    Pre-requisites :
+    1. We need atleast one client node to execute this test case
+    1. creats fs volume create cephfs if the volume is not there
+    2. ceph fs subvolumegroup create <vol_name> <group_name> --pool_layout <data_pool_name>
+        Ex : ceph fs subvolumegroup create cephfs subvol_cross_platform_snapshot_1
+    3. ceph fs subvolume create <vol_name> <subvol_name> [--size <size_in_bytes>] [--group_name <subvol_group_name>]
+       [--pool_layout <data_pool_name>] [--uid <uid>] [--gid <gid>] [--mode <octal_mode>]  [--namespace-isolated]
+       Ex: ceph fs subvolume create cephfs subvol_2 --size 5368706371 --group_name subvolgroup_
+    4. Create Data on the subvolume. We will add known data as we are going to verify the files
+        Ex: create_file_data()
+    5. Create snapshot of the subvolume
+        Ex: ceph fs subvolume snapshot create cephfs subvol_2 snap_1 --group_name subvol_cross_platform_snapshot_1
+
+    Script Flow:
+    1. Mount the subvolume on the client using Kernel and fuse mount
+    2. Write data into the fuse mount point
+    3. collect the c
+    3. Try creating directory and file inside the mount_point/.snap directory(CEPH-83573420)
```

Comment:
Could we remove this step if we are not covering this scenario.

Reply:
Segregating of test cases caused messed up the doc correcting it